# NOTE: Do not use 1e-6 format because this is reqognized as a string

# Framework Settings
pub_on_prop: false
use_ros_time_now: false
verbose: false
verbose_out_of_order: false
discard_ooo_prop_meas: false
pub_cov: true
pub_path: false
buffer_size: 2000

# Ros Settings
use_tcpnodelay: true
bypass_init_service: false

pub_cb_buffer_size: 1
sub_imu_cb_buffer_size: 200
sub_sensor_cb_buffer_size: 1

# Sensor Settings
gyro_rate_noise: 0.426188 # Noise STD [rad/s]
gyro_bias_noise: 0.154667
acc_noise: 0.89791        # Noise STD [m/s^2]
acc_bias_noise: 0.28402

# Core state covariance p,v,q,bw,ba
core_init_cov_p: [0.5,0.5,0.5]
core_init_cov_v: [0.3,0.3,0.3]
core_init_cov_q: [0.1218,0.1218,0.1218] # 20degree
core_init_cov_bw: [0.0076,0.0076,0.0076]
core_init_cov_ba: [0.01,0.01,0.01]

# Pose1 sensor settings
pose1_pos_meas_noise: [ 0.01,0.01,0.01 ]
pose1_rot_meas_noise: [ 0.3,0.3,0.3 ]
pose1_use_dyn_meas_noise: false
pose1_cal_p_ip: [ 0.11713453029, 0.02147913299, -0.06399307175 ]
pose1_cal_q_ip: [ 0.9238795, 0., 0.3826834, 0. ] #w,x,y,z
pose1_state_init_cov: [0.01,0.01,0.01, 0.03,0.03,0.03] #position[3], rot[3]

# Pose2 sensor settings
pose2_pos_meas_noise: [ 0.01,0.01,0.01 ]
pose2_rot_meas_noise: [ 0.3,0.3,0.3 ]
pose2_use_dyn_meas_noise: false
pose2_cal_p_ip: [ 0.019, 0.013, -0.109 ]
pose2_cal_q_ip: [ 0,1,0,0 ] #w,x,y,z
pose2_state_init_cov: [0.01,0.01,0.01, 0.03,0.03,0.03] #position[3], rot[3]

# Position sensor settings
position1_pos_meas_noise: [0.1,0.1,0.1]
position1_use_dyn_meas_noise: false
position1_cal_p_ip: [-0.2009562, 0.0125133, 0.0273]
position1_cal_q_ip: [ 0.5468328, 0, 0, 0.8372418]
position1_state_init_cov: [0.01,0.01,0.01]
